src/main/java/edu/wpi/first/shuffleboard/components/TilePane.java

```diff
@@ -29,7 +31,7 @@
 
   private static final int DEFAULT_COL_COUNT = 1;
   private static final int DEFAULT_ROW_COUNT = 1;
-  private static final double MIN_TILE_SIZE = 32; // pixels
+  private static final double MIN_TILE_SIZE = 64; // pixels
```

Comment:
JavaFX doesn't have a notion of `%` or `em` sizing. Saying this is pixels might be a misnomer because of the scaling that JavaFX does automatically for high DPI displays